test/test_albumtypes.py

```diff
@@ -106,6 +106,6 @@ def _set_config(self, types: [(str, str)], ignore_va: [str], bracket: str):
 
     def _create_album(self, album_types: [str], artist_id: str = 0):
         return self.add_album(
-            albumtypes='; '.join(album_types),
+            albumtypes=album_types,
```

Comment:
and also when creating the test-album we can rely on the `DelimitedString` type to give us a `;`-delimited string already.